Makefile

```diff
@@ -146,12 +146,13 @@ delete-integration-test-dev-cluster: build ## Delete the test cluster for use wh
 ##@ Code Generation
 
 .PHONY: generate-always
-generate-always: ## Generate code (required for every build)
```

Comment:
just want to confirm if you intentionally want to remove this make target details in `make help`.